### DISCUSSION TITLE
What is the highest-grossing holiday movie 🎦  of all time ?

### DISCUSSION THREAD

**Discussion by Sriansh-raj** [Sriansh-raj/GitHub-Achievements#38] [Galaxy Qs] [answered]
What is the highest-grossing holiday movie 🎦  of all time ?

**Comment by lotus647** [comment, marked as answer]
The Grinch?

**Reply by Sriansh-raj** [reply]
Yes it is 😄

**Comment by ChinoBeach-Colgate** [comment]
Home Alone

**Reply by Sriansh-raj** [reply]
Well that's the second highest grossing.

keep tryin 😄

**Comment by NomanGhiasy** [comment]
The Grinch is the highest grossed holiday movie of all time with over 500 Million.

**Reply by lotus647** [reply]
yaaaaa i was right ⬆️⬆️⬆️

**Comment by saurabhcoded** [comment]
The Grinch
